.github/workflows/ci.yml

```
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    name: rake test
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version: ['3.1', '3.0', '2.7']
+        libexif-version: ['system', 'latest']
```

```suggestion
        libexif-version:
          - master
          - libexif-0_6_21-release
          - libexif-0_6_22-release
          - libexif-0_6_23-release
          - libexif-0_6_24-release
```

Would like to make it simpler. See the following review.

---

It will require updating the workflow when there is a new `libexif` version, but this is indeed simpler.

I added a new commit with this change, but I kept the `system` version as well, as I think it is best to test with a distribution-provided package too.

---

> It will require updating the workflow when there is a new libexif version, but this is indeed simpler.

I think it's quite common, especially for Ruby C extension. Just like we need to update `ruby-version` matrix when there is a new Ruby version. And I usually see people using `ruby-version: ['head', '3.1', '3.0', '2.7']` to test the latest Ruby regardless the version number.

> I added a new commit with this change, but I kept the system version as well, as I think it is best to test with a distribution-provided package too.

It's a bit weird to me but I don't know how to say it. However, I don't see any drawback, so let's do as you said, thanks.